<launch>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="bot_joystick" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.2" />
  </node>

 <!-- Axes -->
  <param name="axis_left" value="1" type="int"/>
  <param name="axis_right" value="4" type="int"/>
  <param name="scale_cmd" value="250" type="double"/>
  <node pkg="my_joystick" type="teleop_joy" name="teleop"/>

  <!-- for odom -->
  <param name="min_drive_cmd" value="80" type="int"/>
  <param name="scale_linear_mps" value="0.15" type="double"/>
  <param name="botwidth_m" value="0.15" type="double"/>
</launch>
